server/src/main/java/io/seata/server/coordinator/DefaultCoordinator.java

```diff
@@ -379,9 +379,6 @@ protected void handleRetryRollbacking() {
                     if (ROLLBACK_RETRY_TIMEOUT_UNLOCK_ENABLE) {
                         rollbackingSession.clean();
                     }
-                    // Prevent thread safety issues
```

Comment:
How does this pr fix `TimeoutRollbacking ->TimeoutRollbackRetrying`? I didn't get it.